sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.16.0-beta.1 (Unreleased)
+## 1.16.0 (2021-07-06)
 
```

Comment:
Are there any new features or fixes?

Reply:
Yep, asked @christothes to add a changelog for the feature he added.

Reply:
![](https://media.giphy.com/media/l0Iyl55kTeh71nTXy/giphy.gif)